src/components/Equipments.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {Image, View, Text, StyleSheet} from 'react-native';
+import sword from '../assets/images/icon-sword.png';
+import shield from '../assets/images/icon-shield.png';
+import agility from '../assets/images/icon-agility.png';
+
+export const Equipments = ({item}) => {
```

Comment:
Since this Component is a single item, you could change the name of it to `EquipmentItem` or `EquipmentCard`.

You could also rename the param that this component is receiving. Like `({ item: equipment })`